CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.1.0] - 2021-02-05
```

Comment:
This could be a major version bump (since it has stopped support for JWT version 6), but since version `3.0.0` didn't actually work properly I've just bumped the minor version. Happy to change that if people disagree.

Reply:
I think play it safe and do a major change. Better safe than sorry

Reply:
Changed to `4.0.0` (and also updated the date to Monday's date, since I don't want to risk deploying this one on a Friday evening 😃)